playbook-website/app/views/pages/getting_started_partials/_react_getting_started.md

```diff
@@ -1,34 +1,25 @@
-# React Getting Started
-
 #### Bundle Install
-
 ```sh
 yarn add playbook-ui
 ```
-
 #### Match your project's versions of React and ReactDOM with Playbook's versions
 
 ```json
 "react": "16.8.6",
  "react-dom": "16.8.6",
 ```
-
 #### Import fonts and CSS styles
-
 Can be imported in your Index.js file or top level app Component
 
 ```js
 import 'playbook-ui/dist/fonts/fontawesome-min';
 import 'playbook-ui/dist/fonts/regular-min';
 import 'playbook-ui/dist/playbook.css';
 ```
-
 #### Import Playbook React components
 
 ```js
 import { Avatar, Button } from 'playbook-ui';
 ```
-
 #### CodeSandbox React Setup Example
-
 [Link to CodeSandbox Example](https://codesandbox.io/s/cool-platform-ydfb3)
```

Comment:
All the pages look great! This codesandbox link however is wrong, here's the correct link: https://codesandbox.io/s/playbook-empty-6ixcw

@augustomallmann